golangci-lint/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"strconv"
+)
+
+func main() {
```

Comment:
🚫 **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>other declaration of main (typecheck)